src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -91,9 +115,9 @@
 
     static {
         DESC_LUNCH = new EditEventDescriptorBuilder().withDescription(VALID_DESCRIPTION_LUNCH)
-                .withTime(VALID_TIME_LUNCH).build();
+                .withTime(VALID_TIME_LUNCH).setPersonsToAdd(2, 1).setPersonsToRemove(3, 1).build();
```

Comment:
This would be a descriptor to add persons with index 2 and 1 from the contact list and remove persons 3 and 1 from the attendees list. They are two different things hence its ok. In actual use, if the persons are not found, an error would be displayed.

Reply:
Thanks for clarifying